### PR TITLE
Table and module names differ in 1.7

### PR DESCRIPTION
**Pull request by antoin-m**

### CODE REVIEW

**Comment by maximebiloe** [comment]
Thank you!

File: statsnewsletter.php [hunk 1]
```diff
@@ -42,6 +46,16 @@ public function __construct()
 		$this->author = 'PrestaShop';
 		$this->need_instance = 0;
 
+		if (version_compare(_PS_VERSION_, '1.7.0.0', '>=')) {
+			$this->table_name = _DB_PREFIX_ . 'emailsubscription';
+			$this->newsletter_module_name = 'ps_emailsubscription';
+			$this->newsletter_module_human_readable_name = 'Email subscription';
```
Comment (PrestaEdit) [inline statsnewsletter.php:52]: $this->l(), maybe ?

File: statsnewsletter.php [hunk 2]
```diff
@@ -42,6 +46,16 @@ public function __construct()
...
+		} else {
+			$this->table_name = _DB_PREFIX_ . 'newsletter';
+			$this->newsletter_module_name = 'blocknewsletter';
+			$this->newsletter_module_human_readable_name = 'Newsletter block';
```
Comment (PrestaEdit) [inline statsnewsletter.php:56]: $this->l(), maybe ?

**Review by PrestaEdit — COMMENTED** [review]
I thnk we need translations, there.

File: statsnewsletter.php (hunk 1 above)
Comment (antoin-m) [reply statsnewsletter.php:52]: The complete sentence is translated [here](https://github.com/PrestaShop/statsnewsletter/pull/1/commits/5947dffaca466ef36547a15bea2312e1064aa033#diff-c09969987f5e785ca9a6bf24d64ce6f6R102)

Comment (PrestaEdit) [reply statsnewsletter.php:52]: Mmmmmh. You are right.

It's not the way I usually see (like the use of sprintf) but, right right. If the sentence is seen by the PrestaShop Translation tools.

**Pull request merged**